environment/deployments/data-curation/main.tf

```diff
@@ -138,9 +138,33 @@ module "storage_bucket_3" {
 resource "google_storage_bucket_iam_binding" "hips-vista-bucket-ro-iam-binding" {
   bucket  = module.storage_bucket_3.name
   role    = "roles/storage.objectViewer"
-  members = [
-    "serviceAccount:${var.hips_service_account}"
-  ]
+  members = var.hips_service_accounts
+}
+
+// HiPS Storage Bucket (DP0.2)
+module "storage_bucket_4" {
+  source        = "../../../modules/bucket"
+  project_id    = module.project_factory.project_id
+  storage_class = "REGIONAL"
+  location      = "us-central1"
+  suffix_name   = ["dp02-hips"]
+  prefix_name   = "static"
+  versioning = {
+    dp02-hips = false
+  }
+  force_destroy = {
+    dp02-hips = false
+  }
+  labels = {
+    environment = var.environment
+    application = "hips"
```

Comment:
What does this do?

Reply:
It shows up in the browser:
<img width="613" alt="Screen Shot 2022-06-09 at 16 32 43" src="https://user-images.githubusercontent.com/1054393/172961951-e9423379-f023-4119-ace0-b30e833321a5.png">